SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -178,6 +178,10 @@ lemma mul_eq_and (a b : BitVec 1) : a * b = a &&& b := by
   have hb : b = 0 ∨ b = 1 := width_one_cases _
   rcases ha with h | h <;> (rcases hb with h' | h' <;> (simp [h, h']))
 
+theorem sub_eq_add_neg {w : Nat} {x y : BitVec w} : x - y = x + (- y) := by
```

Comment:
```suggestion
theorem add_neg {w : Nat} {x y : BitVec w} : x + (-y) = x - y := by
```

Check for similar naming in `Int.` or `Nat.` to confirm that this is the canonical form.

Reply:
We probably also want to make this a `@[simp]`.

Reply:
https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Defs.html#sub_eq_add_neg

Reply:
https://github.com/leanprover/lean4/blob/0edf1bac392f7e2fe0266b28b51c498306363a84/src/Init/Data/Int/Lemmas.lean#L87-L87